### PR TITLE
feat: unify code highlight [WPB-12089]

### PR DESCRIPTION
**Pull request by olafsulich**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-12089" title="WPB-12089" target="_blank">StoryWPB-12089</a>  [Web] Text formatting via UI in text input field
  </td></table>
  <br />
 

## Description

Change [Hihglight.js](https://highlightjs.org/) to [Prism.js](https://prismjs.com/) to achieve unified code syntax highlighting in the Lexical editor and an actual message.

I changed the theme, it's more readable now, looks great in both light and dark mode.

### Before

image

image

image

image

### After

image

image

image

image

## Checklist

- [x] mentions the JIRA issue in the PR name (Ex. [WPB-XXXX])
- [x] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

### CODE REVIEW

File: src/script/util/messageRenderer.test.ts [hunk 1]
```diff
@@ -497,59 +497,19 @@ describe('Markdown for code snippets', () => {
...
+</code></pre>`;
 
     expect(
       renderMessage(
         '```xml\n<dependency>\n  <groupId>com.ibm.icu</groupId>\n  <artifactId>icu4j</artifactId>\n  <version>53.1</version>\n</dependency>\n```',
       ),
     ).toEqual(expected);
   });
-
-  it('renders escaped Ruby code blocks', () => {
```
Comment (olafsulich) [inline src/script/util/messageRenderer.test.ts:510]: We shouldn't store those deleted tests, it's the library job

**Review by przemvs — APPROVED** [review]

File: src/script/util/messageRenderer.ts [hunk 2]
```diff
@@ -179,7 +179,12 @@ export const renderMessage = (message: string, selfId?: QualifiedId, mentionEnti
         // highlighting will be wrong anyway because this is not valid code
         return escape(code);
       }
-      return hljs.highlightAuto(code, lang ? [lang] : undefined).value;
+
+      if (lang && languages[lang]) {
+        return highlightCode({code, grammar: languages[lang], lang});
+      }
+
+      return highlightCode({code, grammar: languages.javascript, lang: 'javascript'});
```
Comment (tlebon) [inline src/script/util/messageRenderer.ts:187]: should js be the default?

Comment (olafsulich) [reply src/script/util/messageRenderer.ts:187]: 🤷‍♂️ , I don't have a better idea, do you?

Comment (tlebon) [reply src/script/util/messageRenderer.ts:187]: i think its fine.

**Pull request merged**